solidity/test/LiquidityProtection.js

```diff
@@ -542,7 +585,7 @@ describe('LiquidityProtection', () => {
         await poolToken.approve(liquidityProtection.address, balance);
         await liquidityProtection.protectLiquidity(poolToken.address, '200');
 
-        liquidityProtection.setTime(now.add(duration.seconds(1)));
```

Comment:
Holy c#$%p, I've placed several of those in the code. How did this even work???

Reply:
Not sure why it got removed. I'll restore it

Reply:
Ah, I've replaced them with a global `setTime` that handles other contracts as well

Reply:
Yeah, you actually fixed that here (made me fix it in the other PR, so we'll probably have some merge-conflicts.

Reply:
I've merged and retested everything